api/v1alpha1/drpolicy_types.go

```diff
@@ -81,7 +81,10 @@ type DRPolicySpec struct {
 
 	// The set of managed clusters governed by this policy, which have
 	// replication relationship enabled between them.
-	DRClusterSet []ManagedCluster `json:"drClusterSet"`
+	DRClusterSet []ManagedCluster `json:"drClusterSet,omitempty"`
```

Comment:
Should this line be deleted?

Reply:
Yes, but need to finish the refactor first, for example the second commit does that. This can now be deleted, will add to the task list, but retained in the first pass as otherwise code wont compile.

Reply:
Done